js/onoffice-form-preview.js

```diff
@@ -85,6 +85,17 @@
         }
     });
 
+    document.querySelectorAll('form[data-estate-search-name]').forEach(element => {
```

Comment:
HTML Forms for forms (in terms of configurable forms in the plugin settings) have a different attribute name that depends of the kind of form. This selector would only work for estate search forms.